ReShade.vcxproj

```diff
@@ -397,6 +399,7 @@
     <ClCompile Include="source\windows\ws2_32.cpp" />
   </ItemGroup>
   <ItemGroup>
+    <ClInclude Include="res\d3dx12.h" />
```

Comment:
That shouldn't be here =)

Reply:
Did not know where to put it (remember, it is a WIP version ;-) )

Reply:
In the next, I think good to create a pull request with draft.

Reply:
I know, but it is a WIP version. Where do you want me to put it ?

Reply:
Yes, i will think about it the next time. Do you know how to switch to draft version on github ?